gax/src/main/java/com/google/api/gax/batching/v2/BatchingDescriptor.java

```diff
@@ -79,6 +80,7 @@
  * @param <RequestT> The type of the request that will contain the accumulated elements
  * @param <ResponseT> The type of the response that will be unpacked into individual element results
  */
+@BetaApi("The surface for batching is not stable yet and may change in the future.")
```

Comment:
Please add `@InternalExtensionOnly("For google-cloud-java client use only")`  as well

Reply:
Added `@InternalExtensionOnly` annotation for `BatchingDescriptor` and `RequestBuilder`.